http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -354,6 +357,17 @@ public String clientName() {
          * @see ProxyConfiguration#nonProxyHosts()
          */
         Builder proxyConfiguration(ProxyConfiguration proxyConfiguration);
+
+        /**
+         * Set the {@link TlsKeyManagersProvider} for this client. The {@code KeyManager}s will be used by the client to
+         * authenticate itself with the remote server if necessary when establishing the TLS connection.
+         * <p>
+         * By default the configured provider is {@link SystemPropertyTlsKeyManagersProvider}.
```

Comment:
Does this default behavior have a test?

Reply:
Yes, here https://github.com/aws/aws-sdk-java-v2/blob/4552ea2f7febe6cc0b5e7353bad534b035194fb9/http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/NettyClientTlsAuthTest.java#L134

Reply:
Got it, sorry I missed that.